test/unit/data/schema/integrity_spec.js

```diff
@@ -42,3 +45,62 @@ describe('DB version integrity', function () {
         fixturesHash.should.eql(currentFixturesHash);
     });
 });
+
+/**
+ * @NOTE
+ *
+ * If this test fails for you, you have modified the default settings.
+ * When you make a change or add new setting, please test that:
+ *
+ * 1. A new blog get's installed and the settings looks correct and complete.
+ * 2. A blog get's updated from a lower Ghost version and the settings looks correct and complete.
+ *
+ * Typical cases:
+ * You have to add a migration script to update group/flags if you've added new setting keys.
+ */
+
+describe('Settings integrity', function () {
+    let ghostServer;
+    before(function () {
+        return ghost()
```

Comment:
Unsure if its the right approach to do ghost start in unit tests as we don't have precedence for it, but need to do this here to initialize and test settings in DB, interested in any counter input/thoughts on this or if I missed a better approach. 🤔

Reply:
Yeah, this isn't a unit test if it has to start the whole of Ghost :confused: 

I would either move this test out of the unit tests, OR use a more "unit-ey"  approach of hashing the file and checking it - like we do above for the fixtures & schema

Reply:
Thanks @allouis, have re-written integrity tests as 2 separate checks, one in unit which checks for file hash changes, and other which checks correct values are added to DB.